lib/routes/account.js

```diff
@@ -410,6 +410,8 @@ module.exports = function (
           })
         }
 
+        request.emitMetricsEvent('flow.signin.submit')
```

Comment:
Dope, I thought the discussion in that issue was that we needed to add this event to the route. I'll just nuke the commit.